src/theme.js

```diff
@@ -384,7 +387,7 @@ function getTheme({ style, name }) {
       {
         scope: "markup.list",
         settings: {
-          foreground: primer.yellow[9],
+          foreground: foreground,
```

Comment:
On github.com, only the `*` or `-` are yellow:

![image](https://user-images.githubusercontent.com/378023/85508320-c8b0c400-b62e-11ea-86eb-6de90bc997b9.png)

So I think we could do the same with:

```js
        scope: "punctuation.definition.list.begin.markdown",
        settings: {
          foreground: primer.yellow[9],
        },
```

Reply:
primer.orange[6] looks closer to your screenshot